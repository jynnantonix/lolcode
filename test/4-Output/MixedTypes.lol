This test makes sure the print expression works with various types as arguments.

HAI
	VISIBLE 1 2.345 "Lorem ipsum dolor sit" MKAY?
KTHXBAI
