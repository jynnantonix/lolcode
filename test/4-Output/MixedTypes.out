1 2.345 Lorem ipsum dolor sit 
